openfisca_core/parameters.py

```diff
@@ -378,8 +386,9 @@ def __init__(self, name = "", directory_path = None, data = None, file_path = No
         else:
             self.file_path = file_path
             _validate_parameter(self, data, data_type = dict, allowed_keys = self._allowed_keys)
-            # We allow to set a reference and a description for a node.
```

Comment:
This comment seems outdated (we don't set a reference around it). I'd just remove it 🙂